src/components/ha-sidebar.ts

```diff
@@ -129,6 +133,8 @@ class HaSidebar extends LitElement {
       }
     }
 
+    this._rtl = computeRTL(hass);
```

Comment:
render is not allowed to have side effects. Move to `updated()`

Reply:
Moved